### PR TITLE
Update utils.dart

### PR DESCRIPTION
**Pull request by dejunDev**
fix Issue #101 Bad State

### CODE REVIEW

**Review by srinivasa-b — APPROVED** [review]

**Comment by TinoGuo** [comment]
@dejunDev pls fix the test accordingly, thx

File: lib/src/util/utils.dart [hunk 1]
```diff
@@ -21,12 +21,13 @@ bool overrideDebugWebValue = false;
...
 double platformMiniFontSize() {
-  try {
-    if (isWeb()) {
-      return 1; // Web is not allowed font size less than 1
-    }
-    return double.minPositive;
-  } catch (_) {
-    return 1;
-  }
+  // try {
```
Comment (TinoGuo) [inline lib/src/util/utils.dart]: comment can be removed, since it's broken

**Review by srinivasa-dev — APPROVED** [review]

**Comment by TinoGuo** [comment]
The test is not updated, pls find test file in test folder

**Review by TinoGuo — APPROVED** [review]

**Comment by dejunDev** [comment]
ok. i already fixed it

**Comment by srinivasa-b** [comment]
@TinoGuo Can you make a release on pub.dev with this fix?

**Comment by TinoGuo** [comment]
@srinivasa-b I will do it tonight

**Pull request merged**